src/redux/slices/graphSlice.ts

```diff
@@ -2,9 +2,20 @@ import { DSNPUserId } from "@dsnp/sdk/dist/types/core/identifiers";
 import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 import { GraphChange } from "../../utilities/types";
 
-interface graphState {
-  following: Record<DSNPUserId, Record<DSNPUserId, boolean>>;
-  followers: Record<DSNPUserId, Record<DSNPUserId, boolean>>;
+export enum RelationshipStatus {
+  ESTABlISHED,
+  UPDATING,
+}
+
+export interface graphState {
+  following: Record<DSNPUserId, Record<DSNPUserId, RelationshipStatus>>;
+  followers: Record<DSNPUserId, Record<DSNPUserId, RelationshipStatus>>;
```

Comment:
Hmm... What is the followers RelationshipStatus mean?

Is it the relationship of the follower to the current user or the other way around? The follower to me will always be ESTABLISHED (as otherwise they are not a follower and will not be in the list). The other way around could have a relationship status of... well there isn't one, but `NONE` I guess.

Reply:
The status is the status of the relationship itself.  Updating mostly means the relationship is in flux and the user doesn't get to make further changes until it's resolved one way or the other.